pkg/services/featuremgmt/registry.go

```diff
@@ -1353,6 +1353,13 @@ var (
 			Owner:           grafanaAlertingSquad,
 			RequiresRestart: true,
 		},
+		{
```

Comment:
from line 1238: 
- please add (backend only) in the description
- please replace the owner by `grafanaBackendGroup`

Reply:
Search and Storage